database/sql/query_log.go

```diff
@@ -0,0 +1,279 @@
+package sql
+
+import (
+	"fmt"
+
+	vtparser "github.com/knocknote/vitess-sqlparser/sqlparser"
+	"github.com/pkg/errors"
+	"go.knocknote.io/octillery/exec"
+	"go.knocknote.io/octillery/sqlparser"
+)
+
+// GetParsedQueryByQueryLog get instance of `sqlparser.Query` by QueryLog.
+// If QueryLog has LastInsertID value, add to query it
+func (t *Tx) GetParsedQueryByQueryLog(log *QueryLog) (sqlparser.Query, error) {
+	parser, err := sqlparser.New()
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+	query, err := parser.Parse(log.Query, log.Args...)
+	if err != nil {
+		return nil, errors.WithStack(err)
+	}
+	if query.QueryType() != sqlparser.Insert {
+		return query, nil
+	}
+
+	insertQuery := query.(*sqlparser.InsertQuery)
+	insertQuery.SetNextSequenceID(log.LastInsertID)
+	t.replaceInsertQueryByQueryLog(log, insertQuery)
```

Comment:
Did you forget to handle error?

Reply:
`replaceInsertQueryByQueryLog` always returns `nil` as `error` type.
So, I fixed that interface.